src/dmd/objc.d

```diff
@@ -161,12 +163,11 @@ extern (C++) struct ObjcClassDeclaration
     ClassDeclaration metaclass;
 
     /// List of non-inherited methods.
-    FuncDeclarations* methodList;
+    FuncDeclaration[] methodList;
```

Comment:
Unrelated change

Reply:
I don't think it's unrelated. I changed the type because it's easier to operate on a D array than `Array`.

Reply:
Fair enough, allthough `Array` is used by default in dmd (and it supports `opSlice`).

Reply:
Why do you think that is? Could it possibly have something to do with the fact that DMD used to be written in C++ 😉.